src/app/CommandSender.h

```diff
@@ -436,8 +516,59 @@ class CommandSender final : public Messaging::ExchangeDelegate
 
     CHIP_ERROR SendCommandRequestInternal(const SessionHandle & session, Optional<System::Clock::Timeout> timeout);
 
+    void OnResponseCallback(const ConcreteCommandPath & aPath, const StatusIB & aStatusIB, TLV::TLVReader * apData,
+                            const AdditionalResponseData & aAdditionalResponseData)
+    {
+        if (mUsingExtendedCallbacks)
```

Comment:
Suggest making quick comment mention that "mpCallback" is a legacy path omitted if using extended callbacks.

Reply:
Explicit comment is useful regarding "retired mpCallback" and "recommended mpExtendedCallback"